lib/otel/segments/consumer.js

```diff
@@ -37,9 +38,11 @@ function createConsumerSegment(agent, otelSpan) {
   //   'host',
   //
   // )
-  transaction.name = segmentName
+  transaction.setPartialName(segmentName)
```

Comment:
you don't finalize the name here.  That will be done in the `onEnd` when we end the consumer span's transaction later.  Also the `segmentName` only needs to be `${system}/${destKind}/Named/${destination}`.